src/buffer/buffer_pool_manager.cpp

```diff
@@ -42,13 +42,22 @@ Page *BufferPoolManager::FetchPageImpl(page_id_t page_id) {
   // 2.     If R is dirty, write it back to the disk.
   // 3.     Delete R from the page table and insert P.
   // 4.     Update P's metadata, read in the page content from disk, and then return a pointer to P.
+
+  // For grading. Do not Modify!
+  counter.AddCount(FuncType::FetchPage);
```

Comment:
This should **not** be in the implementation method (`FetchPageImpl`). This should be in the public API method (`FetchPage`).